api/healthcheck/healthcheck_controller.go

```diff
@@ -70,6 +72,9 @@ func (c *controller) aggregate(overallState map[string]h.State) *health.Health {
 	details := make(map[string]interface{})
 	for name, state := range overallState {
 		state.Status = convertStatus(state.Status)
```

Comment:
If we decide to adopt library’s status “ok” and “failed” this change should be propagated everywhere since all the components check UP to determine if it is started correctly.